src/main.rs

```diff
@@ -207,6 +208,26 @@ async fn handle_get_manifest(name: String, reference: String) -> Response {
             .into_response()
     } else {
         let file_path = root.join("blobs").join("sha256").join(&sha_manifest);
+        let media_type_manifest = match fs::read_to_string(file_path.clone())
```

Comment:
Do we need the clone here or could we borrow? I am not positive we need the new PathBuf.

```suggestion
        let media_type_manifest = match fs::read_to_string(&file_path)
```